gpkit/__init__.py

```diff
@@ -24,7 +24,23 @@
 
 # global variable initializations
 DimensionalityError = ValueError
-units = None
+ureg, units = None, None  # pylint: disable=invalid-name
+
+
+class GPkitUnits(object):
+    "Return monomials instead of Quantitites"
+
+    def __init__(self):
+        self.Quantity = ureg.Quantity  # pylint: disable=invalid-name
+        if hasattr(ureg, "__nonzero__"):
```

Comment:
is the `if hasattr` due to changes in pint, or because `ureg` is sometimes not a pint object
? Inline comment explaining please.